docs/combine.rst

```diff
@@ -0,0 +1,59 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plants.
+    :tags: fort productivity stockpiles items plants
+
+Usage
+-----
+
+::
+
+    combine <all|here> [--dryrun] [--types=<comma separated list of types>]
+
+Examples
+--------
+``combine``
+    Displays help
+``combine all --dryrun``
+    Preview stack changes for all types in all stockpiles.
+``combine all``
+    Merge stacks for all stockpile and all types
+``combine all --types=meat,plant``
+    Merge ``meat`` and ``plant`` type stacks in all stockpiles.
+``combine here``
+    Merge stacks in stockpile located at game cursor.
+
+Commands
+-----------------
+``all``
+    Search all stockpiles.
+``here``
+    Search the stockpile under the game cursor.
+
+Options
+-------------
+``--help``
+    Prints help text. Default if no options are specified.
+``--dryrun``
+    Display the stack changes without applying them.
+``--types=<comma separated list of types>``
```

Comment:
```suggestion
``-t``, ``--types <comma separated list of types>``
```

Reply:
the `=` is valid syntax of course, and is fine in the Examples section, but the standard in the docs is to use a space between the option name and its argument in the Options section

Reply:
okay.